app/src/main/java/com/example/javacodersnairobi/service/GithubService.java

```diff
@@ -0,0 +1,16 @@
+package com.example.javacodersnairobi.service;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class GithubService {
+    private Retrofit retrofit;
+    private final String BASE_URL = "http://api.github.com/";
```

Comment:
Good job @ongebo, Don't you have to add the stack and country query strings to the URL?

Reply:
I think that should be added in the interface defining the API endpoints. Consider the endpoints below:

**Get all Users**
`https://api.github.com/search/users?q=type:User+location:Nairobi+language:JAVA%22`
**Get a single user**
`https://api.github.com/users/k33ptoo`

The base URL for the endpoints is: `http://api.github.com/`

Reply:
That's ok, I'll merge this in. @ongebo